spec/factories/tasks.rb

```diff
@@ -1,5 +1,6 @@
 FactoryBot.define do
   factory :task do
-    
+    title {"title1"}
+    status {"todo"}
```

Comment:
enumの値はsymbolを使って定義しましょう
```suggestion
    status { :todo }
```